app/src/main/java/com/owncloud/android/ui/activity/FileDisplayActivity.java

```diff
@@ -1513,24 +1513,28 @@ public void onReceive(Context context, Intent intent) {
             }
 
             boolean uploadWasFine = intent.getBooleanExtra(FileUploadWorker.EXTRA_UPLOAD_RESULT, false);
-            boolean renamedInUpload = getFile().getRemotePath().equals(intent.getStringExtra(FileUploadWorker.EXTRA_OLD_REMOTE_PATH));
 
-            boolean sameFile = getFile().getRemotePath().equals(uploadedRemotePath) || renamedInUpload;
-            Fragment details = getLeftFragment();
+            boolean renamedInUpload = false;
+            boolean sameFile = false;
+            if (getFile() != null) {
+                renamedInUpload = getFile().getRemotePath().equals(intent.getStringExtra(FileUploadWorker.EXTRA_OLD_REMOTE_PATH));
```

Comment:
```suggestion
                renamedInUpload = Objects.equals(getFile().getRemotePath(), intent.getStringExtra(FileUploadWorker.EXTRA_OLD_REMOTE_PATH));
```

Never rely on chaining equals unless you're absolutely sure one if it won't be null